JSDemos/Demos/Tabs/Overview/React/App.tsx

```diff
@@ -43,24 +56,29 @@ const App = () => {
     setIconPosition(e.value);
   }, [setIconPosition]);
 
-  const orientationChanged = React.useCallback(
-    (e) => {
-      setWidgetWrapperClasses(`widget-wrapper widget-wrapper-${e.value}`);
-      setOrientation(e.value);
-    }, [setOrientation, setWidgetWrapperClasses],
-  );
+  const orientationChanged = React.useCallback((e) => {
+    setWidgetWrapperClasses(`widget-wrapper widget-wrapper-${e.value}`);
+    setOrientation(e.value);
+  }, [setOrientation, setWidgetWrapperClasses]);
 
   const showNavigationChanged = React.useCallback((e) => {
+    const shouldRestrictWidth = e.value || scrollContent;
+
+    enforceWidthConstraint(shouldRestrictWidth);
```

Comment:
Isn't it better to get rid of parameters here and just use `scrollContent` and `showNavigation` in `enforceWidthRestriction`?

Reply:
No, in this case scrollContent and showNavigation have old values.